File: group-001/PigLatinTest.php
<?php
require 'PigLatin.php';

/**
 * Created by PhpStorm.
 * User: burned
 * Date: 11.10.16
 * Time: 21:13
 */
class PigLatinTest extends PHPUnit_Framework_TestCase
{
    /**
     * @var PigLatin
     */
    protected $pigLatin;

    protected function setUp()
    {
        $this->pigLatin = new PigLatin();
    }
    
    public function testBeginsWithVowel()
    {
        $this->assertEquals('appleay', $this->pigLatin->piggify('apple'));
    }

    public function testBeginsWithConsonant()
    {
        $this->assertEquals('irdbay', $this->pigLatin->piggify('bird'));
        $this->assertEquals('ancay', $this->pigLatin->piggify('can'));
    }

    public function testCompleteSentence()
    {
        $this->assertEquals('aay ellowyay irdbay', $this->pigLatin->piggify('a yellow bird'));
    }

    public function testConsonantsClusters()
    {
        $this->assertEquals('airchay', $this->pigLatin->piggify('chair'));
        $this->assertEquals('aresquay', $this->pigLatin->piggify('square'));
    }
}

File: group-005/src/test/java/PigLatinTest.java
import org.junit.Test;

import static junit.framework.TestCase.assertEquals;

/**
 * Created by tweinreich on 11.10.16.
 */
public class PigLatinTest {

    @Test
    public void appendAyWhenStartsWithVowelsTest() {
        String testWord = "apple";
        String expectedOutput = "appleay";
        PigLatin pigLatin = new PigLatin();
        assertEquals(expectedOutput, pigLatin.translate(testWord));
    }

    @Test
    public void wordStartsWithConsonantTest() {
        String testWord = "bird";
        String expectedOutput = "irdbay";
        PigLatin pigLatin = new PigLatin();
        assertEquals(expectedOutput, pigLatin.translate(testWord));
    }

    @Test
    public void phraseTest() {
        String testPhrase = "a yellow bird";
        String expectedOutput = "aay ellowyay irdbay";
        PigLatin pigLatin = new PigLatin();
        assertEquals(expectedOutput, pigLatin.translate(testPhrase));
    }

    @Test
    public void consonantClusterTest() {
        String testWord = "chair";
        String expectedOutput = "airchay";
        PigLatin pigLatin = new PigLatin();
        assertEquals(expectedOutput, pigLatin.translate(testWord));
    }
}
